pkg/controllers/consolidation/controller.go

```diff
@@ -456,7 +474,10 @@ func (c *Controller) nodeConsolidationOptionReplaceOrDelete(ctx context.Context,
 			schedulableCount += len(inflight.Pods)
 		}
 		if len(node.pods) == schedulableCount {
-			savings := node.instanceType.Price()
+			savings, err := getNodePrice(node)
+			if err != nil {
+				return consolidationAction{result: consolidateResultUnknown}, fmt.Errorf("getting offering price from candidate node, %w", err)
```

Comment:
This deviates from the standard go error return pattern. Typically, the `foo, err` pattern is an OR semantic. Either you get a value OR you get an error. The pattern is typically to return the uninitialized type value if there's an error. For example: nil, 0, "", or in this case, consolidationAction{}.

Reply:
This is consistent with how it was written in other places throughout the code when an error is returned. At this point, it looks like this error is being logged at the level above but then the action is what is being used to determine what to do with the consolidation reconciliation loop so it looks like the error isn't passed all the way up

Reply:
huh, ok.

Reply:
In this case it's the same thing, consolidateResultUnknown = 0